README.md

```diff
@@ -5,7 +5,7 @@
    <p>The Swiss Army Knife<br/>for the SIGHUP Distribution</p>
 
    [![Build Status](https://ci.sighup.io/api/badges/sighupio/furyctl/status.svg?ref=refs/heads/main)](https://ci.sighup.io/sighupio/furyctl)
-   ![Release](https://img.shields.io/badge/furyctl-v0.32.4-blue)
+   ![Release](https://img.shields.io/badge/furyctl-v0.32.5-blue)
```

Comment:
Why is this showing up? May be this branch needs a rebase?

Reply:
Fixed with a new rebase